test/api/v3/integration/tasks/POST-create_task.test.js

```diff
@@ -0,0 +1,67 @@
+import {
+  generateUser,
+  requester,
+  translate as t,
+} from '../../../../helpers/api-integration.helper';
+import { v4 as generateRandomUserName } from 'uuid';
+import { each } from 'lodash';
+
+describe('POST /tasks', () => {
+  let user;
+  let api;
+
+  before(() => {
+    return generateUser().then((generatedUser) => {
+      user = generatedUser;
+      api = requester(user);
+    });
+  });
+
+  context('checks "req.body.type"', () => {
+    it('returns an error if req.body.type is absent', () => {
+      expect(api.post('/tasks', {
```

Comment:
This expect call is using chai-as-promised and needs to be returned to work properly. (otherwise mocha might finish the test before checking the expectation)

It's an async test, and the way to do that in Mocha is either to include a callback or to return a promise. One of the things that chai-as-promised does is turn the expect function into a promise.